database/mssql/scripts/versions/revert/v_41_ddl_revert.sql

```diff
@@ -0,0 +1,44 @@
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+SET NOCOUNT ON
+GO
+
+SET XACT_ABORT ON
+
+BEGIN TRY
+  BEGIN TRANSACTION
+      ALTER TABLE [permit].[ORBC_LOA_PERMIT_TYPE_DETAILS] DROP CONSTRAINT FK_ORBC_LOA_PERMIT_TYPES_PERMIT_TYPE
+      ALTER TABLE [permit].[ORBC_LOA_DETAILS] DROP CONSTRAINT FK_ORBC_LOA_DETAILS_PREVIOUS_LOA_ID
+      ALTER TABLE [permit].[ORBC_LOA_DETAILS] DROP CONSTRAINT FK_ORBC_LOA_DETAILS_ORIGINAL_LOA_ID
+      ALTER TABLE [permit].[ORBC_LOA_DETAILS] DROP COLUMN PREVIOUS_LOA_ID
+      ALTER TABLE [permit].[ORBC_LOA_DETAILS] DROP COLUMN ORIGINAL_LOA_ID
+      ALTER TABLE [permit].[ORBC_LOA_DETAILS] ALTER COLUMN [START_DATE] [varchar](10) NOT NULL
+      ALTER TABLE [permit].[ORBC_LOA_DETAILS] ALTER COLUMN [EXPIRY_DATE] [varchar](10) NULL
+      EXEC sp_rename @objname = 'permit.ORBC_LOA_PERMIT_TYPE_DETAILS.PERMIT_TYPE', @newname = 'PERMIT_TYPE_ID', @objtype = 'COLUMN'
+      ALTER TABLE [permit].[ORBC_LOA_PERMIT_TYPE_DETAILS] DROP CONSTRAINT [FK_ORBC_LOA_PERMIT_TYPE_LOA_ID]
+      ALTER TABLE [permit].[ORBC_LOA_VEHICLES] DROP CONSTRAINT [FK_ORBC_LOA_VEHICLES_LOA_ID]
+      ALTER TABLE [permit].[ORBC_LOA_PERMIT_TYPE_DETAILS]
+      WITH CHECK ADD CONSTRAINT [ORBC_LOA_PERMIT_TYPES_PERMIT_TYPE_ID_FK] FOREIGN KEY ([PERMIT_TYPE_ID]) REFERENCES [permit].[ORBC_PERMIT_TYPE]([PERMIT_TYPE])
+      ALTER TABLE [permit].[ORBC_LOA_PERMIT_TYPE_DETAILS] CHECK CONSTRAINT [ORBC_LOA_PERMIT_TYPES_PERMIT_TYPE_ID_FK]
+      ALTER TABLE [permit].[ORBC_LOA_VEHICLES]
+      WITH CHECK ADD CONSTRAINT [FK_ORBC_LOA_VEHICLES_LOA_ID] FOREIGN KEY ([LOA_ID]) REFERENCES [permit].[ORBC_LOA_DETAILS]([LOA_ID])  ON DELETE CASCADE
+      ALTER TABLE [permit].[ORBC_LOA_VEHICLES] CHECK CONSTRAINT [FK_ORBC_LOA_VEHICLES_LOA_ID]
+      ALTER TABLE [permit].[ORBC_LOA_PERMIT_TYPE_DETAILS]
+      WITH CHECK ADD CONSTRAINT [FK_ORBC_LOA_PERMIT_TYPE_LOA_ID] FOREIGN KEY ([LOA_ID]) REFERENCES [permit].[ORBC_LOA_DETAILS]([LOA_ID])  ON DELETE CASCADE
```

Comment:
Its ORBC_LOA_PERMIT_TYPE_LOA_ID_FK in v31. Please cross reference names with v31 and confirm

Reply:
done